### DISCUSSION TITLE
Request - Remember Playback Speed per User

### DISCUSSION THREAD

**Discussion by pavichokche** [jellyfin/jellyfin-androidtv#3022] [General]
My wife likes to watch her shows on 1.25 speed or higher - AWESOME feature. But when we switch to the kids' account we always need to switch the speed off (if we notice...poor kids). Then when she wants to watch her shows again it needs to be turned on again. Simple quality of life fix, should be fairly simple to implement.

**Comment by nielsvanvelzen** [comment]
The app should clear the playback speed when it is reopened. If switching the user does not clear it I'd say that is a bug.

**Comment by pavichokche** [comment]
Alright, so I tested it and this is certainly not working as it should.
Switching users, backing out of the app to the home screen (using the Back button) - nothing makes the playback speed reset to default UNLESS you 'Force Stop' the whole app.
